Output all on one line with version 0.10.1

It seems like there was a change from 0.10.0 to 0.10.1 where the output from commands gets written all on a single line. For example:

```
ubuntu@037d120b6d55:/workspaces/mmwebsite (main)$ /tmp/precious-0.10.0 --version
precious 0.10.0
ubuntu@037d120b6d55:/workspaces/mmwebsite (main)$ /tmp/precious-0.10.0 lint -g
💍 Linting modified files according to git
💯 Passed git-diff-check: go.mod
💯 Passed git-diff-check: go/pkg/geoip-build/snowflake/db.go
💥 error go-mod-tidy: go.mod
💯 Passed golangci-lint-fmt: go/pkg/geoip-build/snowflake/db.go
💥 error golangci-lint: go/pkg/geoip-build/snowflake/db.go
Errors when linting files:
  ▶ [commands.go-mod-tidy] failed for [go.mod]
    Failed to execute command `go mod tidy -diff`: Got unexpected exit code 1 from `go mod tidy -diff`.
Stdout:
diff current/go.mod tidy/go.mod
--- current/go.mod
+++ tidy/go.mod
@@ -13,13 +13,11 @@
        aidanwoods.dev/go-paseto v1.6.0
        cloud.google.com/go v0.123.0
        cloud.google.com/go/apikeys v1.2.7
-
        cloud.google.com/go/auth v0.18.0
        cloud.google.com/go/bigquery v1.72.0
        cloud.google.com/go/compute v1.53.0
        cloud.google.com/go/compute/metadata v0.9.0
        cloud.google.com/go/iam v1.5.3
-
        cloud.google.com/go/logging v1.13.1
        cloud.google.com/go/monitoring v1.24.3
        cloud.google.com/go/profiler v0.4.3


Stderr was empty.

  ▶ [commands.golangci-lint] failed for [go/pkg/geoip-build/snowflake/db.go]
    Failed to execute command `/workspaces/mmwebsite/dev/bin/installed/golangci-lint run go/pkg/geoip-build/snowflake`: Got unexpected exit code 1 from `/workspaces/mmwebsite/dev/bin/installed/golangci-lint run go/pkg/geoip-build/snowflake`.
Stdout:
go/pkg/geoip-build/snowflake/db.go:27:1: exported: exported function FromFlags should have comment or be unexported (revive)
func FromFlags() (*app.App, *sql.DB, error) {
^
1 issues:
* revive: 1

Stderr was empty.

ubuntu@037d120b6d55:/workspaces/mmwebsite (main)$ /tmp/precious-0.10.1 --version
precious 0.10.1
ubuntu@037d120b6d55:/workspaces/mmwebsite (main)$ /tmp/precious-0.10.1 lint -g
💍 Linting modified files according to git
💯 Passed git-diff-check: go.mod
💯 Passed git-diff-check: go/pkg/geoip-build/snowflake/db.go
💥 error go-mod-tidy: go.mod
💯 Passed golangci-lint-fmt: go/pkg/geoip-build/snowflake/db.go
💥 error golangci-lint: go/pkg/geoip-build/snowflake/db.go
"\u{1b}[31mErrors when linting files:\u{1b}[0m\n  ▶ [commands.go-mod-tidy] failed for [go.mod]\n    Lint command \"go-mod-tidy\" failed to execute: Failed to execute command `go mod tidy -diff`: Got unexpected exit code 1 from `go mod tidy -diff`.\nStdout:\ndiff current/go.mod tidy/go.mod\n--- current/go.mod\n+++ tidy/go.mod\n@@ -13,13 +13,11 @@\n \taidanwoods.dev/go-paseto v1.6.0\n \tcloud.google.com/go v0.123.0\n \tcloud.google.com/go/apikeys v1.2.7\n-\n \tcloud.google.com/go/auth v0.18.0\n \tcloud.google.com/go/bigquery v1.72.0\n \tcloud.google.com/go/compute v1.53.0\n \tcloud.google.com/go/compute/metadata v0.9.0\n \tcloud.google.com/go/iam v1.5.3\n-\n \tcloud.google.com/go/logging v1.13.1\n \tcloud.google.com/go/monitoring v1.24.3\n \tcloud.google.com/go/profiler v0.4.3\n\n\nStderr was empty.\n\n  ▶ [commands.golangci-lint] failed for [go/pkg/geoip-build/snowflake/db.go]\n    Lint command \"golangci-lint\" failed to execute: Failed to execute command `/workspaces/mmwebsite/dev/bin/installed/golangci-lint run go/pkg/geoip-build/snowflake`: Got unexpected exit code 1 from `/workspaces/mmwebsite/dev/bin/installed/golangci-lint run go/pkg/geoip-build/snowflake`.\nStdout:\ngo/pkg/geoip-build/snowflake/db.go:27:1: exported: exported function FromFlags should have comment or be unexported (revive)\nfunc FromFlags() (*app.App, *sql.DB, error) {\n^\n1 issues:\n* revive: 1\n\nStderr was empty.\n\n"ubuntu@0
```

What do you think?

Thank you!

--- a/Changes.md
+++ b/Changes.md
@@ -1,3 +1,8 @@
+## 0.10.2
+
+- Fixed a bug where error output was printed with debug formatting, which caused escaped ANSI codes
+  and newlines to appear in the output. GH #90.
+
 ## 0.10.1 2025-10-27
 
 - Fixed a bug when calling `precious -c precious.toml`, without a directory in the path to the

--- a/precious-core/src/precious.rs
+++ b/precious-core/src/precious.rs
@@ -521,7 +521,7 @@ impl LintOrTidyRunner {
             Ok(e) => {
                 debug!("{e:?}");
                 if let Some(e) = e.error {
-                    print!("{e:?}");
+                    print!("{e}");
                 }
                 if let Some(msg) = e.message {
                     println!("{} {}", self.chars.empty, msg);

--- a/precious-integration/src/lint_tidy.rs
+++ b/precious-integration/src/lint_tidy.rs
@@ -414,6 +414,42 @@ lint-failure-exit-codes = 1
 
 #[test]
 #[serial]
+fn output_on_failure() -> Result<()> {
+    let helper = set_up_for_tests()?;
+    helper.write_file("src/bad.rs", "this is not valid rust")?;
+
+    let precious = precious_path()?;
+
+    let out = Exec::builder()
+        .exe(&precious)
+        .args(vec!["lint", "--all"])
+        .ok_exit_codes(&[1])
+        .in_dir(&helper.precious_root())
+        .build()
+        .run()?;
+
+    let stdout = out.stdout.unwrap_or_default();
+    let stderr = out.stderr.unwrap_or_default();
+    let output = format!("{}{}", stdout, stderr);
+
+    assert!(
+        !output.contains(r"\u{1b}"),
+        "Output should not contain escaped ANSI codes"
+    );
+    assert!(
+        !output.contains(r"\n"),
+        "Output should not contain escaped newlines"
+    );
+    assert!(
+        output.contains("linting files:"),
+        "Output should contain the error header"
+    );
+
+    Ok(())
+}
+
+#[test]
+#[serial]
 fn all_invocation_options() -> Result<()> {
     let helper = set_up_for_tests()?;
     write_perl_script(&helper)?;

